README.md

```diff
@@ -0,0 +1,96 @@
+<div align="center">

```

Comment:
We've resisted renaming `README` to `README.md` for a long time because we wanted it to be readable as a text file. It was anticipated that as soon as it's a Markdown file, we'll start getting requests to improve it visually at the expense of that basic textual clarity. The beauty of markdown is that it can be read in both modes as long as you keep it simple. This HTML header would be a start in an undesirable direction. If there's a nice way to do with this kind of header framing and styling without intruding on the README.md text content, we should do that, but we probably wouldn't be into adding straight-up HTML to the file unless our attitude about this document has shifted.

Reply:
so che changes wont be merged...